### ISSUE TITLE
中文分类训练精度只有0.33

### ISSUE THREAD

**Issue by ruisunyc** [brightmart/albert_zh#116]
使用albert_config_tiny预训练模型进行下游中文分类任务，训练结果只有0.33的精度
训练结果如下：
INFO:tensorflow:***** Eval results E:\pythonProject\albert_zh\output3\model.ckpt-1462 *****
INFO:tensorflow:  eval_accuracy = 0.33299127
INFO:tensorflow:  eval_loss = 1.1318309
INFO:tensorflow:  global_step = 1462
INFO:tensorflow:  loss = 1.1323752

参数配置如下
--task_name=news
--do_train=true
--do_eval=true
--do_predict=false
--data_dir=E:\pythonProject\albert_zh\data
--vocab_file=E:\pythonProject\albert_zh\albert_tiny_489k/vocab.txt
--bert_config_file=E:\pythonProject\albert_zh\albert_tiny_489k/albert_config_tiny.json
--init_checkpoint=E:\pythonProject\albert_zh\albert_tiny_489k/albert_model.ckpt
--max_seq_length=256
--train_batch_size=32
--learning_rate=2e-5
--num_train_epochs=3
--output_dir=E:\pythonProject\albert_zh\output3

共三个类别，每个类别6500篇中文，按8：1：1进行分割的

**Comment by brightmart** [comment]
应该是没有训练好吧。你可以使用learning_rate=1e-4, num_train_epochs=6试一试效果。

**Comment by brightmart** [comment]
效果如何？

**Comment by ruisunyc** [comment]
> 效果如何？
还是那，效果不行
--max_seq_length=256
--train_batch_size=32
--learning_rate=1e-4
--num_train_epochs=6

INFO:tensorflow:  eval_accuracy = 0.34889686
INFO:tensorflow:  eval_loss = 1.1013632
INFO:tensorflow:  global_step = 1
INFO:tensorflow:  loss = 1.1012403

**Comment by brightmart** [comment]
发到群里给确定一下。你这个看上去不是算法问题，是你的代码问题，因为这个成绩是随机猜的分数，也就是模型还没有学。

**Comment by brightmart** [comment]
由于是其他模型也是这个精度，所以可能是输入和输出没有设置对
